### DISCUSSION TITLE
How to update a parameter?

### DISCUSSION THREAD

**Discussion by AndiRudi** [microsoft/aspire#6122] [Q&A] [answered]
Now I added some parameters to my aspire application ... in my case the stripe secret. When I first issued `azd up` it asked me to enter the secret and I did... but for some reason I entered the wrong secret. How can I now change it? I tried to change it in the app in azure but it always gets overwritten with a new deployment. Any ideas are appreciated...

**Comment by vhvb1989** [comment, marked as answer]
Look for the file config.json. It should be inside the folder .azure/{your-env-name}/config.json

Then find the parameter in the infra.parameters section and delete it. (It should've been referencing a local vault, you don't need to worry about that).

Then run `azd up` again and azd will ask you for the value again.

These steps would keep your current deployment in Azure and just re-prompt for the parameter. Another option is to run `azd down` to delete your deployment from Azure, then run `azd env new` to create a new local azd env and finally `azd up`.
